src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: { name, age, sex, isMarried, partnerName },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {name}</h2>
+    {age && <p className="Person__age">I am {age}</p>}
+    {isMarried ? (
+      <p className="Person__partner">
+        {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
+      </p>
+    ) : (
```

Comment:
better to avoid nested ternary operators 

decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```

Reply:
Thank you! I updated my .jsx file.